blog/2024-02-27-deprecate-bind-hass-and-hass-components.md

```diff
@@ -0,0 +1,92 @@
+---
+author: Jan-Philipp Benecke
+authorURL: https://github.com/jpbede
+authorImageURL: https://avatars.githubusercontent.com/u/3989428?s=96&v=4
+title: "Deprecate use of @bind_hass and hass.components"
+---
+
+As of Home Assistant 2024.3, we deprecate the use of the `@bind_hass` decorator 
+and thus also the use of `hass.components`.
+With the release of Home Assistant 2024.3, we are starting to adjust and remove usage in Home Assistant Core.
+Authors of custom integrations will need to adjust their code to avoid breakage.
```

Comment:
We should update this message to explain that it will keep working until 2024.9 but that we advise custom integrations to adjust already now.

Reply:
Updated 👍